client/my-sites/domains/domain-management/name-servers/custom-nameservers-form.jsx

```diff
@@ -132,9 +108,7 @@ class CustomNameserversForm extends React.PureComponent {
 
 		return (
 			<Card compact className="name-servers__custom-nameservers-form">
-				<span>{ translate( 'Use Custom Name Servers:' ) }</span>
-
-				{ this.warning() }
+				<strong>{ translate( 'Use custom name servers:' ) }</strong>
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.